samples/highcharts/demo/styled-mode-dumbbell/demo.css

```diff
@@ -0,0 +1,43 @@
+@import 'https://code.highcharts.com/css/highcharts.css';
+
+#container {
+	height: 400px;
+	max-width: 800px;
+	margin: 0 auto;
+}
+
+/* Link the series colors to axis colors */
+.highcharts-color-0.highcharts-lollipop-low {
+	fill: #90ed7d;
+	stroke: #90ed7d;
+}
+.highcharts-color-0.highcharts-lollipop-high {
+	fill: #7cb5ec;
+	stroke: #7cb5ec;
+}
+.highcharts-axis.highcharts-color-0 .highcharts-axis-line {
+	stroke: #7cb5ec;
+}
+.highcharts-axis.highcharts-color-0 text {
+	fill: #7cb5ec;
+}
+.highcharts-color-1.highcharts-lollipop-low {
+	fill: #f7a35c;
+	stroke: #f7a35c;
+}
+.highcharts-color-1.highcharts-lollipop-high {
+	fill: #434348;
+	stroke: #434348;
+}
+.highcharts-axis.highcharts-color-1 .highcharts-axis-line {
+	stroke: #90ed7d;
+}
+.highcharts-axis.highcharts-color-1 text {
+	fill: #90ed7d;
+}
+.highcharts-yaxis .highcharts-axis-line {
+	stroke-width: 2px;
+}
+.highcharts-lollipop-stem {
+	stroke: #f45b5b;
+}
```

Comment:
Shouldn't we move all of these styles to `highcharts.scss` file? @TorsteinHonsi any thoughts?

Reply:
In many demos there is this kind of solution. In `highcharts.scss` there are more general styles.